plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py

```diff
@@ -189,7 +197,7 @@ def estimate_priority_fee(
         [
             reward[0]
             for reward in fee_history.get("reward", [])
-            if reward[0] >= min_allowed_tip
+            if reward[0] >= min_allowed_tip.get(web3_object.eth.chain_id, 0)
```

Comment:
:muscle:

Reply:
Maybe the config will require changes:

https://github.com/valory-xyz/open-aea/blob/f3f323394f99c8fcdda46bf753a875beac46a619/packages/valory/connections/ledger/connection.yaml#L40